CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Bug Fixes
 
+- Updated `_steps_util.py` to throw a specific exception when drive cycle starts at t>0
```

Comment:
```suggestion
- Updated `_steps_util.py` to throw a specific exception when drive cycle starts at t>0 ([#3756](https://github.com/pybamm-team/PyBaMM/pull/3756))
```
Linking PRs helps us keep track in changelog 🙂

Reply:
@arjxn-py Thanks I missed that when I reviewed

Reply:
@R-Yash I merged this so the tests could start

Reply:
Thanks @kratman, marking this as resolved